Tests/SonarScanner.MSBuild.Test/AnalysisWarning/NetFrameworkBootstrapperClassTests.cs

```diff
@@ -73,7 +73,7 @@ public void Exe_PostProc_SucceedsAndWarningCreated()
                 mockFrameworkVersionProvider.Setup(x => x.IsOlderThan462FrameworkVersion()).Returns(true);
```

Comment:
naming consistency: `IsOlderThan462FrameworkVersion` vs `lowerThan462FrameworkVersion` - I would suggest to keep only one term (old vs low)

Reply:
I opted for lower.